Build/run.sh

```diff
@@ -56,8 +56,9 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 CONTNAME="$(whoami)-rzv2l_vlp_v3.0.0${BRANCH}"
-#Create OUTDIR if it doesn't exist
+#Create OUTDIR if it doesn't exist and chmod it to 777
 mkdir -p ${OUTDIR}
+chmod 777 ${OUTDIR}
```

Comment:
let's combine the two commands.

    mkdir -p -m 777 ${OUTDIR}

Reply:
Prime! It's obvious but I didn't know about the mkdir `-m` flag! Great!

Reply:
Probklem is if the directory already exists and doesn't have the mode, `mkdir -p -m 777 test` doesn't set the mode! I prefer to keep it as two separate lines TBH! Especially because we're dealing with people that may have run this script already.... if it was no patch but implementation from day 1, I agree `-p -m` would be the right thing to do.